packages/react-native/ReactCommon/React-Fabric.podspec

```diff
@@ -47,6 +47,19 @@ Pod::Spec.new do |s|
   s.dependency "RCTTypeSafety", version
   s.dependency "ReactCommon/turbomodule/core", version
   s.dependency "React-jsi", version
+  s.dependency "React-logger"
+  s.dependency "glog"
+  s.dependency "DoubleConversion"
+  s.dependency "React-Core"
+  s.dependency "React-debug"
+  s.dependency "React-utils"
+  s.dependency "React-runtimescheduler"
+
+  if ENV["USE_HERMES"] == nil || ENV["USE_HERMES"] == "1"
```

Comment:
Might need to flip this since Hermes isn't enabled by default for us

Reply:
There are a number of other places that test `ENV['USE_HERMES'] == nil || ENV['USE_HERMES'] == '1'`. I'm wondering if it makes sense to refactor this everywhere.

Reply:
Oh. If it's working up until now, I think it's because we define the end variable at the podfile, so it's always either 0 or 1. So we're good